particles/Native/Wasm/source/TestParticle.kt

```diff
@@ -117,7 +117,7 @@ class TestParticle : Particle() {
             info.clear()
         }
         eventHandler("throw") {
-            throw Exception("this message doesn't get passed (yet?)")
+            throw kotlin.Exception("this message doesn't get passed (yet?)")
```

Comment:
Interesting that qualifying the exception with `kotlin.` is required..

Reply:
IntelliJ said it was an error unless I put this. :(

Reply:
I think the move is to import exception from Kotlin at the top of the file, then use `Exception` without the `kotlin` qualifier.